guides/back-end-projects.md

```diff
@@ -58,8 +58,17 @@ For testing we prefer using [pytest](https://docs.pytest.org/), following its co
 
 However you may find the use of unittest in some (older) projects.
 
-Code coverage is normally not checked.
-Absolute code coverage requirements and enforcement thereof exist only in very rare cases.
+In earlier projects code coverage was normally not checked.
+Absolute code coverage requirements and enforcement thereof in these projects exist only in very rare cases.
+
+In newer projects code coverage is however normally checked. Absolute coverage requirements are part of these projects.
```

Comment:
Perhaps add `see the README of the project for more information` or something similar to indicate that different projects have different requirements and that those can be found there.